spec/admin.mediawiki

```diff
@@ -45,4 +45,25 @@ possible.
 
 See also [[fundamentals.mediawiki/#Exchange_Variables|Exchange Variables]].
 
-...
+==Administration API==
+
+The server will provide an HTTP API for performing various adminstrative tasks.
+
+'''API Endpoints'''
```

Comment:
API Endpoints look kind of funky for me. I think the list should continue down? But config has its own row, then everything else is on the second row:
![endpoints](https://user-images.githubusercontent.com/34998433/84741838-a0034b80-afea-11ea-91d4-890df05ff71b.png)